HW1/P2/P2b.py

```diff
@@ -1,3 +1,32 @@
+import findspark 
+findspark.init()
 from P2 import *
+from pyspark import SparkContext
+import random
 
-# Your code here
+sc = SparkContext()
+
+#create lists of the range for rows and cols  
+ilist = range(2000)
+jlist = range(2000)
+
+random.shuffle(ilist)
+random.shuffle(jlist)
```

Comment:
@jacobbaron is right. Why don't you do the shuffle inside Spark? For example, you can call a partitionBy with a shuffle on the index on the rdd mandelrdd. Then you won't have any issue if the data set is too large. And also that way the work is even more balanced.